test/test_psutil.py

```diff
@@ -2475,6 +2475,25 @@ def test_Popen(self):
             proc.wait()
             self.assertIsNotNone(proc.returncode)
 
+    @unittest.skipUnless(hasattr(psutil.Process, "environ"),
+                         "environ not available")
+    def test_environ(self):
+        self.maxDiff = None
```

Comment:
https://docs.python.org/2/library/unittest.html#unittest.TestCase.maxDiff 
interesting; didn't know about this